school-finder-frontend/app/components/nav/NavController.js

```diff
@@ -0,0 +1,13 @@
+(function () {
+  var sfApp = angular.module('sfApp');
+
+  var NavCtrl = function($scope, $location){
+      $scope.isActive = function (viewLocation) {
+          return viewLocation === $location.path().replace(/^(?:\/)+/, "");;
```

Comment:
@DevinClark Good catch! I was hoping there was a better way to determine the active route. The PR has been updated.